docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,102 @@
+# Convert ONNX* RNNT to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction covers conversion of RNN-t model from [MLCommons](https://github.com/mlcommons) repository. Before
+converting into IR PyTorch model need to be exported into ONNX. In order to that follow the steps below.
+
+**Step 1**. Clone RNN-t PyTorch implementation from MLCommons repository. In order to pull only RNN-t model 
+without full repository make shallow clone. If you alreade have a full repository skip this and go to **Step 2**:
+```bash
+git clone -n https://github.com/mlcommons/inference rnnt_for_openvino --depth 1
+cd rnnt_for_openvino
+git checkout HEAD speech_recognition/rnnt 
+```
+
+**Step 2**. If you already have a full clone of MLCommons inference repository create a folder for 
+pre-trained PyTorch model and where conversion into IR will take place. Skip this step if you did a shallow clone.
+Also further you will need to specify path your full clone:
+```bash
+mkdir rnnt_for_openvino 
+cd rnnt_for_openvino
+```
+
+**Step 3**. Download pre-trained weights for PyTorch implementation from https://zenodo.org/record/3662521#.YG21DugzZaQ:
+For UNIX* like systems you can use wget
+```bash
+wget https://zenodo.org/record/3662521/files/DistributedDataParallel_1576581068.9962234-epoch-100.pt
+```
+The link was taken from `setup.sh` in the `speech_recoginitin/rnnt` subfolder. You will get exactly the same weights as 
+if you have done steps from https://github.com/mlcommons/inference/tree/master/speech_recognition/rnnt
+
+**Step 4**. Install required python packages:
+```bash
+pip install torch toml
+```
```

Comment:
`pip install torch toml` is command line needed to install packages in Windows, but `pip3 install torch toml` is command line in Linux.

Reply:
Indded `pip` works on Linux as well (as well as `pip3` works on Windows). The only confusion can happen if there is also Python 2 in the system `pip` can call Python 2 instead of expected Python 3. For the sake of unification a changed everywhere to pip3/python3.